packages/core-react/package.json

```diff
@@ -9,21 +9,26 @@
     "dev": "yarn storybook",
     "build": "gulp scripts:es scripts:umd scripts:compiled",
     "test": "jest",
+    "tdd": "jest --watch",
```

Comment:
nit: `test:watch`

Reply:
I'm the only one who prefers `tdd`, huh?

Reply:
personally I usually pass the flags in the yarn commands not super opinionated about this though